pkg/log/log.go

```diff
@@ -36,26 +38,44 @@ type GlobalConfig struct {
 }
 
 func init() {
-	zapCfg := zap.NewDevelopmentConfig()
-	zapCfg.EncoderConfig.EncodeLevel = zapcore.CapitalColorLevelEncoder
-	zapCfg.Level.SetLevel(zap.InfoLevel)
+	zapCfg := newZapConfigWithInfoLevel()
 	l, err := zapCfg.Build()
 	if err != nil {
 		log.Println("Failed to init zap global logger, no zap log will be shown till zap is properly initialized: ", err)
 		return
 	}
+	consensusZapCfg := newZapConfigWithInfoLevel()
+	_consensusLogger, err = consensusZapCfg.Build()
+	if err != nil {
+		log.Println("Failed to init consensus zap logger, no zap log will be shown till zap is properly initialized: ", err)
+		return
+	}
 	_globalCfgMu.Lock()
 	_globalCfg.Zap = &zapCfg
+	_consensusLogCfg.Zap = &consensusZapCfg
 	_globalCfgMu.Unlock()
 	zap.ReplaceGlobals(l)
 }
 
+func newZapConfigWithInfoLevel() zap.Config {
+	zapCfg := zap.NewDevelopmentConfig()
+	zapCfg.EncoderConfig.EncodeLevel = zapcore.CapitalColorLevelEncoder
+	zapCfg.Level.SetLevel(zap.InfoLevel)
+
+	return zapCfg
+}
+
 // L wraps zap.L().
 func L() *zap.Logger { return zap.L() }
 
 // S wraps zap.S().
 func S() *zap.SugaredLogger { return zap.S() }
 
+// C returns consensus log
+func C() *zap.Logger {
```

Comment:
`L` and `S` stands for `Logger` and `SugaredLogger` they are two different way to use global logger.
I don't think consensus logger is same level of things with these two